### ISSUE TITLE
Calling a callable property results in "Unexpected token"

### ISSUE THREAD

**Issue by MidnightDesign** [phpmd/phpmd#549]
The following syntax (`($this->bar)()`) is valid (at least in PHP 7.1), but an error is emitted my PHPMD.

```php
class Foo {
    /** @var callable */
    private $bar;
    public function baz($something) {
        ($this->bar)($something); // Unexpected token: ,, line: ..., col: ..., file: ...
    }
}
```

**Comment by vikkio88** [comment]
same here +1

**Comment by adjenks** [comment]
I also encountered this.

**Comment by jandreasn** [comment]
This is really annoying because we can't suppress an unexpected token error.

**Comment by kylekatarnls** [comment]
Will be fixed via #383

**Comment by kylekatarnls** [comment]
Note, than you can test this development using `"phpmd/phpmd": "dev-php-7.1-compatibility"`
